tsl/src/compression/compression_scankey.c

```diff
@@ -322,10 +322,12 @@ build_index_scankeys_using_slot(Oid hypertable_relid, Relation in_rel, Relation
 			const NameData *attname = attnumAttName(in_rel, in_attnum);
 			AttrNumber column_attno = get_attnum(out_rel->rd_id, NameStr(*attname));
 
-			/* Make sure we find columns in key columns in order to select the right index */
+			/* Make sure we find columns in key columns in order to select the right index
+			 * We skip over any non-key columns
```

Comment:
We're looping over the index keys, so this comment is confusing because it is not clear if it "key columns" refers to index keys or constraint keys. Can we make more clear?

Reply:
Renamed the variable and updated the comment to make more clear which column is which.